libcrypt.map.in

```diff
@@ -19,6 +19,10 @@ crypt_gensalt_ra	XCRYPT_2.0	GLIBC_2.0:owl:suse	GLIBC_2.2.2:alt     OW_CRYPT_1.0:
 # Actively supported interfaces from libxcrypt.
 crypt_checksalt		XCRYPT_4.3
 
+# Interface for code compatibility with older versions
+# (v3.1.1 and earlier) of libxcrypt.
+crypt_gensalt_r		XCRYPT_2.0
```

Comment:
Does this need to be 
```
crypt_gensalt_r	XCRYPT_2.0	GLIBC_2.0:owl:suse	GLIBC_2.2.1:alt     OW_CRYPT_1.0:suse
```
instead?

Reply:
I don't think so, bet me check this quickly…

Reply:
Nope, neither of them used it.  It was a libxcrypt only thing.

Reply:
This implementation is not intended to keep binary compatibility, anyways.  It is simply to keep applications using this interface building without further changes.

Sadly a plain `#define crypt_gensalt_r crypt_gensalt_rn` is not enough, as I've seen configure scripts checking for symbol availability of `crypt_gensalt_r` in `crypt, xcrypt`…

Reply:
@zackw Anything else to consider?